build/DownloadCFamilyPlugin/DownloadCFamilyPlugin.csproj

```diff
@@ -47,14 +20,14 @@
 
   <!-- Building the project creates an assembly that contains this custom task.   
        We then need to execute the CopyEmbeddedFiles task to execute the task and copy the files. -->
-  <UsingTask TaskName="DownloadAndExtract" AssemblyFile="$(OutputPath)DownloadCFamilyPlugin.dll" />
-  
-  <Target Name="CopyEmbeddedFiles" >
-    <Message Importance="high" Text="Fetching CFamily files..." />
 
-    <!-- Parameter validation -->
```

Comment:
I extracted the parameter validation to a separate target. It's only a couple of lines, so I'm not sure it was worth it.